ord_schema/validations.py

```diff
@@ -3,10 +3,17 @@
 import math
 import re
 import warnings
+from urllib import request
+from urllib.error import HTTPError
```

Comment:
Should be `from urllib import error` (and then `error.HTTPError`). If you don't like `error`, you could probably just do `urllib.error.HTTPError`.

Reply:
Done. I was worried about ```error``` but keeping ```urllib.error.HTTPError``` is good